src/ngraph_builder.cc

```diff
@@ -3937,6 +3937,45 @@ static Status TranslateTileOp(
   return Status::OK();
 }
 
+// Translate TopKV2 Op using ngraph core op TopK
+static Status TranslateTopKV2Op(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map) {
+  shared_ptr<ngraph::Node> ng_input;
+  ValidateInputCount(op, 2);
```

Comment:
Is this for debugging purpose? Or we still need this check here.

Reply:
I added while i was debugging but just left it since it is good to validate.